tests/functional/healthchecks/test/checkRoutes.js

```diff
@@ -8,10 +8,11 @@ const async = require('async');
 const Redis = require('ioredis');
 
 const conf = require('../../config.json');
+const { config } = require('../../../../lib/Config');
 
 const redis = new Redis({
-    host: conf.localCache.host,
-    port: conf.localCache.port,
+    host: config.localCache.host || conf.localCache.host,
+    port: config.localCache.port || conf.localCache.port,
```

Comment:
that is a good catch, but we should `only` read from config?


```suggestion
    host: config.localCache.host
    port: config.localCache.port
```

Reply:
or maybe this was by design, to ensure the tests run with the right config.json file (and thus fully reproductible) : and not sure `Config` will pick the right config file?

Reply:
It picks the default config location that shoud be the one from the root of the repo which has the same value.

It's for Integration mainly that it's interesting to rely on the lib/Config rather than a hardcoded test config as we might provide other values